api/rds.proto

```diff
@@ -48,7 +56,7 @@ message WeightedCluster {
   message ClusterWeight {
     // Name of the upstream cluster. The cluster must exist in the cluster
     // manager configuration.
-    google.protobuf.StringValue name = 1;
+    string name = 1;
```

Comment:
Is there a downside to sticking with wrapper types for all scalar values? E.g.. is it going to be less efficient? I'm not a proto3 expert, but https://developers.google.com/protocol-buffers/docs/proto3#default claims that all scalar values (which seem to include strings) don't allow distinction between the default value and not being set. Is the documentation not being clear here?